flask_rebar/compat.py

```diff
@@ -4,71 +4,43 @@
 import pkg_resources
 
 import marshmallow
-
+from marshmallow import ValidationError
 
 if int(sys.version_info[0]) == 2:
     Mapping = collections.Mapping
 else:
     Mapping = collections.abc.Mapping
 
-
 if sys.version_info >= (3, 4):
     ABC = abc.ABC
 else:
     ABC = abc.ABCMeta("ABC", (), {})
 
-
 MARSHMALLOW_DISTRIBUTION = pkg_resources.get_distribution("marshmallow")
 MARSHMALLOW_MAJOR_VERSION = int(MARSHMALLOW_DISTRIBUTION.version.split(".")[0])
-MARSHMALLOW_V2 = MARSHMALLOW_MAJOR_VERSION == 2
 MARSHMALLOW_V3 = MARSHMALLOW_MAJOR_VERSION == 3
```

Comment:
As all this file does is handle compatibility between marshmallow 2 and 3, and 2 is now dropped, you can get rid of this file. Instead of calling compat.load, compat.set_data_key, etc, just call load or set_data_key directly on the field/schema/etc.